### PR TITLE
Replace TravisCI with GitHub Actions

### PR DESCRIPTION
**Pull request by FerPerales**

### CODE REVIEW

File: .github/workflows/rspec.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: RSpec
+
+on:
+  push:
+    branches:
+    - "main"
+  pull_request:
+    branches:
+      - main
```
Comment (mxie) [inline .github/workflows/rspec.yml:9]: For consistency, would you mind fixing the indentation for the branch listing as well as omitting the quotes?

File: .github/workflows/rspec.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Install dependencies
+        run: bundle install
+
+      - name: Run test and linter
+        run: |
```
Comment (mxie) [inline .github/workflows/rspec.yml:27]: The `|` here is good for multi-line commands. In this case, I think we can inline that one line and skip the pipe altogether.

**Review by mxie — COMMENTED** [review]
Thanks for this! Left a few suggestions for you.

**Comment by FerPerales** [comment]
@mxie thanks! Updated

**Review by mxie — APPROVED** [review]
LGTM - thanks!

**Pull request merged**